webclient/app/src/app/features/apps/appCard/AppCard.jsx

```diff
@@ -55,20 +39,21 @@ function AppCard(props) {
                             </Typography>
                         </Grid>
                         <Grid item xs={5} align="right">
-                                <IconButton onClick={onEditClick}><Edit fontSize={"small"} /></IconButton>
-                                <IconButton onClick={() => setOpenDeleteDialog(true)}><Delete fontSize={"small"} /></IconButton>
- 
+                            <IconButton onClick={onEditClick}><Edit fontSize={"small"}/></IconButton>
+                            <IconButton onClick={() => setOpenDeleteDialog(true)}><Delete
+                                fontSize={"small"}/></IconButton>
+
                         </Grid>
                     </Grid>
                 </CardContent>
-                <Divider />
+                <Divider/>
                 <CardActionArea onClick={() => history.push("/apps/" + app.id)}>
                     <CardContent>
                         <Typography variant="body2" className={appCardStyles.description}>
                             {app.packageName}
                         </Typography>
                         <CardActions className={appCardStyles.cardActions}>
-                            <MoreHoriz className={appCardStyles.cardActions} color="primary" />
```

Comment:
Why this long-time formatting standard has changed?

Reply:
My IDE does this formatting forever...